addon/templates/components/drag-sort-list.hbs

```diff
@@ -8,6 +8,7 @@
     tagName         = childTagName
     draggingEnabled = draggingEnabled
     dragEndAction   = dragEndAction
+	disableSorting  = disableSorting
```

Comment:
Wrong indentation type.

Please setup your editor to use http://editorconfig.org/#download .